src/DssSpell.sol

```diff
@@ -16,441 +16,58 @@
-            vow: MCD_VOW,
-            spotter: MCD_SPOT,
-            cat: MCD_CAT,
-            dog: MCD_DOG,
-            end: MCD_END,
-            esm: MCD_ESM,
-            flipperMom: FLIPPER_MOM,
-            clipperMom: CLIPPER_MOM,
-            ilkRegistry: ILK_REGISTRY,
-            pip: DssExecLib.getChangelogAddress("PIP_UNIV2WBTCDAI"),
-            clipper: MCD_CLIP_UNIV2WBTCDAI_A,
-            flipper: DssExecLib.getChangelogAddress("MCD_FLIP_UNIV2WBTCDAI_A"),
-            calc: MCD_CLIP_CALC_UNIV2WBTCDAI_A,
-            hole: 5_000_000 * RAD,
-            chop: 113 * WAD / 100,
-            buf: 115 * RAY / 100,
-            tail: 215 minutes,
-            cusp: 60 * RAY / 100,
-            chip: 1 * WAD / 1000,
-            tip: 0,
-            cut: 995 * RAY / 1000,
-            step: 125 seconds,
-            tolerance: 70 * RAY / 100,
-            clipKey: "MCD_CLIP_UNIV2WBTCDAI_A",
-            calcKey: "MCD_CLIP_CALC_UNIV2WBTCDAI_A",
-            flipKey: "MCD_FLIP_UNIV2WBTCDAI_A"
-        }));
+    // MIP40c3-SP10: Modify Core Unit Budget
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/c23fd23b340ecf66a16f0e1ecfe7b55a5232864d/MIP40/MIP40c3-Subproposals/MIP40c3-SP10.md -q -O - 2> /dev/null)"
+    string constant public MIP40c3SP10 = "0xa3afb63a4710cb30ad67082cdbb8156a11b315cadb251bfe6af7732c08303aa6";
 
-        // --------------------------------- UNIV2WBTCETH-A ---------------------------------
-        flipperToClipper(Collateral({
-            ilk: "UNIV2WBTCETH-A",
-            vat: MCD_VAT,
-            vow: MCD_VOW,
-            spotter: MCD_SPOT,
-            cat: MCD_CAT,
-            dog: MCD_DOG,
-            end: MCD_END,
-            esm: MCD_ESM,
-            flipperMom: FLIPPER_MOM,
-            clipperMom: CLIPPER_MOM,
-            ilkRegistry: ILK_REGISTRY,
-            pip: DssExecLib.getChangelogAddress("PIP_UNIV2WBTCETH"),
-            clipper: MCD_CLIP_UNIV2WBTCETH_A,
-            flipper: DssExecLib.getChangelogAddress("MCD_FLIP_UNIV2WBTCETH_A"),
-            calc: MCD_CLIP_CALC_UNIV2WBTCETH_A,
-            hole: 5_000_000 * RAD,
-            chop: 113 * WAD / 100,
-            buf: 130 * RAY / 100,
-            tail: 200 minutes,
-            cusp: 40 * RAY / 100,
-            chip: 1 * WAD / 1000,
-            tip: 0,
-            cut: 99 * RAY / 100,
-            step: 130 seconds,
-            tolerance: 50 * RAY / 100,
-            clipKey: "MCD_CLIP_UNIV2WBTCETH_A",
-            calcKey: "MCD_CLIP_CALC_UNIV2WBTCETH_A",
-            flipKey: "MCD_FLIP_UNIV2WBTCETH_A"
-        }));
+    // MIP41c4: Facilitator Onboarding (Subproposal Process) Template
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/mips/c23fd23b340ecf66a16f0e1ecfe7b55a5232864d/MIP41/MIP41c4-Subproposals/MIP41c4-SP10.md -q -O - 2> /dev/null)"
+    string constant public MIP41c4 = "0xe37c37e3ffc8a2c638500f05f179b1d07d00e5aa35ae37ac88a1e10d43e77728";
```

Comment:
This mip should be named as follow:
- `// MIP41c4-SP10: Facilitator Onboarding (Subproposal Process) Template`
- `string constant public MIP41c4SP10 = "...";`

Reply:
Fixed in the spell, the github will be fixed but won't be changed in the spell.